src/engine/LinearExpression.h

```diff
@@ -0,0 +1,47 @@
+/*********************                                                        */
+/*! \file LinearExpression.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Guy Katz, Haoze (Andrew) Wu
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+
+ **/
+
+#ifndef __LinearExpression_h__
+#define __LinearExpression_h__
+
+#include "Map.h"
+
```

Comment:
Is this class different from the Equation class? Can we use the existing one?

Reply:
The difference is that linear expression is the lhs of an equation plus a constant: A**x** +b

Reply:
Okay. I believe an equation is Ax = b, so it could also be used, right?
Just trying to reduce code duplication

Reply:
Never mind, I take my comment back. I think the linear expression has a sufficiently different meaning